### ISSUE TITLE
NTP servers blocked: false positive

### ISSUE THREAD

**Issue by lightswitch05** [biroloter/Mobile-Ad-Hosts#1]
Hello,

It looks like several NTP servers have made it onto your block list:

- `2.android.pool.ntp.org`
- `asia.pool.ntp.org`
- `north-america.pool.ntp.org`
- `pool.ntp.org`

NTP is the [Network Time Protocol](https://en.wikipedia.org/wiki/Network_Time_Protocol) and is used solely for synchronizing time across devices.

Since time is very difficult to synchronize across a network with variable latency like the internet, the pool relies on DNS Anycast to select time servers as close to the user as possible to reduce latency. This is why you see generic entries like `pool.ntp.org` that can then resolve to specific areas like `north-america.pool.ntp.org` or `asia.pool.ntp.org`

NTP pools are a free volunteer project - and therefore has limited resources. When large vendors point their devices and software at the generic servers, it can cause a lot of stress on the servers. To avoid this and ensure vendors contribute servers to the pool, NTP requires vendors to create their [vendor specific domains](https://www.ntppool.org/en/vendors.html#vendor-zone), this is why you see entries like `2.android.pool.ntp.org`. Netgear and D-Link have been particularly bad about mistreating the free NTP servers. [NTP server misuse and abuse](https://en.wikipedia.org/wiki/NTP_server_misuse_and_abuse#Notable_cases) is pretty interesting read if you have the time

**Comment by DRSDavidSoft** [comment]
Yes, the `pool.ntp.org` and others should not be blocked, as they do not serve any "ads" whatsoever.

**Comment by p1r473** [comment]
+1 for removing pool.ntp.org

**Comment by yubiuser** [comment]
Why is this still not fixed?

**Comment by p1r473** [comment]
@biroloter Please fix this
